### PR TITLE
feature/2392/org UI fixes

### PR DESCRIPTION
**Pull request by NatalieEO**
dockstore/dockstore#2392
Stargazers visually consistent for tools/workflows/organizations
Made stargazer images smaller
remove redundant titles
don't show info on my starred organizations if info null

Screen Shot 2019-05-14 at 9 36 13 AM
Screen Shot 2019-05-14 at 9 35 19 AM

### CODE REVIEW

**Comment by coverbeck** [comment]
Could you please attach a screenshot of how it looks now?

**Comment by NatalieEO** [comment]
@coverbeck attached images

**Comment by coverbeck** [comment]
I think the Organization Stargazers page should show the organization in question, like it does for a workflow/tool. There are other elements on the page too (last modified, the starring component, that should probably match as well). I think the most important one is the organization name.

In a perfect world, they would be the same component and you would get this for free.

If you want to track this separately in a separate ticket, that's fine.

BTW, I'm not wild about the Back to details button, but we have that pattern already for tools and workflows.

**Comment by coverbeck** [comment]
Also on line 21 of stargarzers.component.html, it will say there are no gazers for this entry instead of organization.

**Comment by NatalieEO** [comment]
Screen Shot 2019-05-16 at 11 23 33 AM

File: src/app/organizations/organization/organization.component.ts [hunk 1]
```diff
@@ -84,4 +85,8 @@ export class OrganizationComponent implements OnInit {
   organizationStarGazersChange(): void {
     this.organizationStarGazersClicked = !this.organizationStarGazersClicked;
   }
+
+  getTimeAgoMessage(date: Date) {
```
Comment (coverbeck) [inline src/app/organizations/organization/organization.component.ts]: Gary comments whenever I do this -- how often is this invoked?  Probably a lot. It's preferable not to invoke methods for display values from your html.

If it's invoked a lot, you may want to either make this a pipe, or perhaps have the OrganizationQuery return it, although I'm not sure if that's a good practice.

**Pull request merged**